libs/dao/src/main/java/com/akto/dto/test_editor/Repository.java

```diff
@@ -0,0 +1,45 @@
+package com.akto.dto.test_editor;
+
+public class Repository {
+    String name;
+    String branch;
+
+    public Repository() {
+    }
+
+    public Repository(String name, String branch) {
+        this.name = name;
+        this.branch = branch;
+    }
```

Comment:
This isn't a repository.

Reply:
can you help with a reference for schema for repository.

Reply:
Updated PR to accomodate for test library zip links instead of github repos and use `content-length` to verify large file sizes. @ankush-jain-akto